ProjFS.Mac/PrjFSKext/PrjFSKext/PerformanceTracing.cpp

```diff
@@ -0,0 +1,61 @@
+#include "PerformanceTracing.hpp"
+#include <sys/types.h>
+#include <stdatomic.h>
+#include <IOKit/IOUserClient.h>
+
+PerfTracingProbe profile_probes[Probe_Count];
+
+void PerfTracing_Init()
+{
+    for (size_t i = 0; i < Probe_Count; ++i)
```

Comment:
Should `PerfTracing_Init` be a no-op when `PRJFS_PERFORMANCE_TRACING_ENABLE` is not defined?

Reply:
I've been trying to minimise the amount of conditionally compiled code as @sanoursa was uneasy about having too much of it due to the potential of it going stale if it's not well exercised. Given the Init functions are only called once, this seemed a relatively benign one to always allow to run.